easybuild/easyconfigs/s/SPPARKS/SPPARKS-07192016.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = "Binary"
+
+name = 'SPPARKS'
+version = '07192016'
+
+homepage = "http://dna.engr.uconn.edu/?page_id=25"
+description = """SPPARKS is a Monte Carlo code that has 
+algorithms for kinetic Monte Carlo (KMC), rejection KMC (rKMC), 
+and Metropolis Monte Carlo (MMC). On-lattice and off-lattice 
+applications with spatial sites on which "events" occur can be 
+simulated in parallel."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+# You need to manually get the source file from http://www.sandia.gov/~sjplimp/download.html
+sources = ['spparks.tar.gz']
+
+checksums = ['e872813f3c4734b360d4ca9acab3e238']
+
+install_cmd = "tar xvf spparks.tar.gz &&"
+install_cmd += "cd %(builddir)s/spparks-19Jul16/src/ &&"
+install_cmd += "make linux &&"
+install_cmd += "mkdir ../bin && cp spk_linux ../bin &&"
+install_cmd += "cp -a %(builddir)s/spparks-19Jul16/* %(installdir)s"
```

Comment:
@rtripath89 it looks like `MakeCp` would be a better option here, combined with use of `buildopts`